core/src/test/java/knaccc/monero/address/WalletAddressTest.java

```diff
@@ -46,5 +47,24 @@ public void testWalletAddress() throws WalletAddress.InvalidWalletAddressExcepti
         assertEquals(walletAddress.getSubaddressBase58(privateViewKeyHex, 0, 1), addr01);
         assertEquals(walletAddress.getSubaddressBase58(privateViewKeyHex, 1, 0), addr10);
         assertEquals(walletAddress.getSubaddressBase58(privateViewKeyHex, 1, 1), addr11);
+
+        assertEquals(walletAddress.checkPrivateViewKey(privateViewKeyHex), true);
+        assertEquals(WalletAddress.arePubPrivKeysRelated(publicViewKeyHex, privateViewKeyHex), true);
+        assertEquals(WalletAddress.arePubPrivKeysRelated(privateViewKeyHex, privateViewKeyHex), false);
+
+        assertEquals(WalletAddress.arePubPrivKeysRelated(
```

Comment:
Better use assertTrue

Reply:
Also package of test should match the tested class (bisq.core.xmr.knaccc.monero.address)